charts/eno-ws/values.yaml

```diff
@@ -0,0 +1,172 @@
+# Default values for eno-ws.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+enoxml:
+  enabled: true
+  replicaCount: 1
+
+  image:
+    repository: inseefr/eno-ws
```

Comment:
maybe `inseefr/eno-ws:2.4.1` here and `inseefr/eno-ws:latest` in enojava (later on)